### ISSUE TITLE
Forecyt and .iex files

### ISSUE THREAD

**Issue by ad05bzag** [RGLab/CytoML#62]
Hi all,

i was wondering if anybody worked with .iex files generated by Intellicyt (https://intellicyt.com/products/instrumentation/). The files are containers that have FCS3.0 format fcs files, gating info is provided in XMLs. Now, their XMLs are not GatingML compliant. 

So, before starting to write a parser I was wondering if anyone worked on similar / that very same problem.
bulat

**Comment by mikejiang** [comment]
No, we have not yet seen this kind of iex files and you are the first person asking about it. So feel free to fork cytoml and add `Intellicyt _to_gatingset` module  if you decide to parse it into `GatingSet`.
